stix_shifter_modules/onelogin/stix_translation/json/from_stix_map.json

```diff
@@ -0,0 +1,61 @@
+{
+  "user-account": {
+    "fields": {
+      "user_id": ["user_id"],
+      "display_name": ["user_name"]
+    }
+  },
+  "ipv4-addr": {
+    "fields": {
+      "value": ["ipaddr"]
+    }
+  },
+
+  "x-onelogin-finding": {
+    "fields": {
+      "client_id": ["client_id"],
+      "directory_id": ["directory_id"],
+      "event_type_id": ["event_type_id"],
+      "unique_id": ["id"],
+      "resolution": ["resolution"],
+      "since": ["since"],
+      "until": ["until"],
+      "proxy_ip": ["proxy_ip"],
+      "policy_id": ["policy_id"],
+      "policy_name": ["policy_name"],
+      "role_id": ["role_id"],
+      "app_id": ["app_id"],
+      "group_id": ["group_id"],
+      "otp_device_id": ["otp_device_id"],
+      "actor_system": ["actor_system"],
+      "custom_message": ["custom_message"],
+      "role_name": ["role_name"],
+      "app_name": ["app_name"],
+      "group_name": ["group_name"],
+      "otp_device_name": ["otp_device_name"],
+      "operation_name": ["operation_name"],
+      "directory_sync_run_id": ["directory_sync_run_id"],
+      "resource_type_id": ["resource_type_id"],
+      "browser_fingerprint": ["browser_fingerprint"],
+      "notes": ["notes"],
+      "created_at": ["created_at"]
+    }
+  },
+  "x-onelogin-risk": {
+    "fields": {
+      "risk_score": ["risk_score"],
+      "risk_reasons": ["risk_reasons"],
+      "risk_cookie_id": ["risk_cookie_id"],
+      "error_description": ["error_description"]
+    }
+  },
+
+  "x-onelogin-user-account": {
```

Comment:
There is already an official STIX `user-account` object. Any properties that don't fit with the existing user-account object and be added as custom properties (denoted by an `x_` prefix) on the official STIX object. Ex: `user-account:x_actor_user_id`

Reply:
Yes. @delliott90 
Resolved.